gcloud/_helpers.py

```diff
@@ -19,23 +19,20 @@
 import calendar
 import datetime
 import os
-import six
+from threading import local as Local
 import socket
 
-try:
-    from threading import local as Local
-except ImportError:     # pragma: NO COVER (who doesn't have it?)
-    class Local(object):
-        """Placeholder for non-threaded applications."""
-
+import six
 from six.moves.http_client import HTTPConnection  # pylint: disable=F0401
 
+from gcloud.environment_vars import PROJECT
+
+# pylint: disable=wrong-import-position
```

Comment:
I am 50-50 on it. I like when it complains that we have third-party and stdlib imports out-of-order. I don't like when it fails to recognize these `try/except` imports are part of imports.